components/fxa-client/src/lib.rs

```diff
@@ -32,8 +32,8 @@ pub mod device;
 pub mod error;
 pub mod ffi;
 pub mod migrator;
+pub mod testing;
 
-#[cfg(feature = "integration_test")]
 pub mod auth;
```

Comment:
Since it's no longer gated on a feature, perhaps we should move this whole `auth` module under the `testing` namespace to make it clear that it should only be used for testing purposes.